client/public/index.html

```diff
@@ -23,26 +23,39 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+<<<<<<< HEAD
+  <title>Ingredient Tracker</title>
```

Comment:
are there two page titles now?

Reply:
apparently yes but not sure why, didn't even open the index.html for this 😆

Reply:
maybe after "git merge main" you said to keep both?

Reply:
yes most likely.. I always do the accept incoming changes or whatever so it probably added that second one.

Reply:
alright, it is definitely good to always check :D